content/src/cloud/remote-ui/remote-ui-deep-dive.md

```diff
@@ -0,0 +1,49 @@
+---
+zendesk:
+  article_id: 25619268678557
+  name: Remote UI - deep dive
+  position: 3
+  labels: cloud
+---
+
+Home Assistant Remote UI allows you to connect to your Home Assistant instance from outside of your home network. By default, remote access is not enabled. You need to [enable it](/hc/en-us/articles/26474279202973/).
+
+Once enabled, Home Assistant generates a security certificate for secure communication and provides you with a URL. You can then use this URL to access your Home Assistant while away from home.
+
+## How it works
+
+This section dives into more technical details on how the remote connection works. If you are only interested in a high level overview, refer to the topic [about the Remote UI connection to Home Assistant](/hc/en-us/articles/26469707849629/).
+
+### Remote UI communication and routing
+
+The Remote UI ensures secure communication between your browser and your local Home Assistant instance by encrypting all data. This encryption is achieved using a Let's Encrypt certificate. Behind the scenes, your local Home Assistant instance connects to one of our custom-built UI proxy servers. These servers operate at the TCP level, forwarding all encrypted data to the local instance.
```

Comment:
forwarding all encrypted data between your browser/client and the local instance.

Reply:
https://github.com/NabuCasa/support/pull/362